noir/Earthfile

```diff
@@ -201,6 +201,42 @@ build-acir-tests:
     RUN ./rebuild.sh true
     SAVE ARTIFACT /usr/src/noir-repo/test_programs/acir_artifacts/*
 
+barretenberg-acir-benches-bb:
+  FROM ../build-images/+build
```

Comment:
note including all of build-images/+build is quite big for an exported image, though it will work. If this can be FROM ubuntu:noble, it's better. I am thinking of adding a +test-base that has everything one might need